packages/react-ui/components/CurrencyInput/CurrencyHelper.tsx

```diff
@@ -23,7 +23,7 @@ export class CurrencyHelper {
   public static defaultOptions: DecimalFormattingOptionsInternal = {
     fractionDigits: null,
     hideTrailingZeros: false,
-    thousandsDelimiter: String.fromCharCode(0x2009),
+    thousandsDelimiter: String.fromCharCode(0x0020),
```

Comment:
@ErmolaevID обычный пробел ожидаемо шире тонкого во всех браузерах. 

![image](https://user-images.githubusercontent.com/4607770/127266567-5906776c-6048-48de-a380-39999fe25150.png)


Думаю, дизайнеры не зря выбрали именно тонкий, и не стоит его тотально менять.  Но в качестве фикса только для Safari можно.

```suggestion
    thousandsDelimiter: String.fromCharCode(isSafari ? 0x0020 : 0x2009),
```


У нас есть готовые флаги для определения браузеров в `packages/react-ui/lib/client.ts`.